includes/gateways/paypal/checkout-actions.php

```diff
@@ -196,6 +196,17 @@ function create_order( $purchase_data ) {
 			$response = $api->make_request( 'v2/checkout/orders', $order_data );
 
 			if ( ! isset( $response->id ) && _is_item_total_mismatch( $response ) ) {
+
+				edd_record_gateway_error(
+					__( 'PayPal Gateway Error', 'easy-digital-downloads' ),
```

Comment:
```suggestion
					__( 'PayPal Gateway Warning', 'easy-digital-downloads' ),
```

Just to clarify that the whole process did not fail.